Code/Tools/FBuild/FBuildCore/Cache/LightCache.h

```diff
@@ -24,7 +24,6 @@ class LightCache
 {
 public:
     LightCache();
-    ~LightCache();
```

Comment:
`~LightCache() = default; // Destruction heavy-lifting is currently done by IncludedFile::~IncludedFile()` ?

Not sure how strongly I feel, since source control should have the history of where this went. And it's not my call anyway -- but I do tend to prefer explicit "there's not supposed to be a destructor right now, because X" kind of message to the implicit which could always be misinterpreted by the reader.

Again, the implicit is that hey, none of these pointers are currently owned by the `LightCache`, so it doesn't need a destructor. But C++ is complex enough that someone could still read the code and be left unsure.

Anyway, this is an op-ed comment that you can safely ignore, since I'm not an owner of this project =)

Reply:
Probably this needs a bit of refactoring to do it properly, the ownership is not very clear right now. Maybe using smart pointers would make it a bit more clear but this is a no stl kind of project.

Having said that, I could add the explicit destructor and the comment. This code is hard to follow as is